source/nodejs/adaptivecards-site/themes/adaptivecards/layout/archive.ejs

```diff
@@ -22,7 +64,7 @@ layout: layout
 	<% if(i % 3 === 0) { /* start a new row every 3 posts */ %>
 	<div class="w3-cell-row w3-padding-16">
 		<% } %>
-		<article class="w3-cell w3-margin-bottom w3-padding-16 w3-container w3-mobile article" style="width: 328px">
+		<article class="w3-cell w3-margin-bottom w3-padding-16 w3-container w3-mobile article" style="width: 328px" onkeyup="onKeyUp(event)" tabindex="0">
```

Comment:
I wasn't quite sure how we should handle this, but I assigned tab order to the main article element so that it can receive focus like so:
![image](https://github.com/microsoft/AdaptiveCards/assets/98650930/8fda6cce-3e7c-4c63-9438-5c204379b25f)

It isn't a clickable element, but it will help jump through the blog posts more quickly.

Reply:
how hard would it be to make it invokable? might be nice :)

Reply:
I _think_ it should be reasonable :)

Reply:
Okay I made it so that you can navigate to the article when you click enter. I didn't want to expose the onclick - that seemed like too much noise